chanzuckerberg/single-cell-data-portal
# fix(wmg-v2): Fixing Dynamic X-Axis Height Logic

## Reason for Change

Slack conversation: https://czi-sci.slack.com/archives/C0247APK621/p1691428500493189

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/109984998/ae1bf4d9-9783-4943-b3db-fd5da28afd78)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/109984998/af0bd89b-b2ba-483a-a245-8ce091750d8a)

Also fixes PNG export Cell Count not being in right location: 
![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/109984998/528da3ff-32f2-45ac-9112-9863f27761a9)


## Changes

- add
- remove
- modify
    - Change in logic for dynamic x-axis height

## Testing steps

Flow 1
1. Select RTEL1-TNFRSF6B gene
2. Select CFH gene
3. Remove RTEL1-TNFRSF6B gene
4. Ensure that x-axis height shrinks to CFH height

Flow 2
1. Select RTEL1-TNFRSF6B gene
2. Select CFH gene
3. Remove CFH gene
5. Ensure that x-axis height stays the same and matches RTEL1-TNFRSF6B height

## Notes for Reviewer

[rdev](https://dynamic-x-axis-frontend.rdev.single-cell.czi.technology)
